Middleware/DynamicJavaScriptMiddleware.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.IO;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Hosting;
+using System.Threading.Tasks;
+
+public class DynamicJavaScriptMiddleware
+{
+    private readonly RequestDelegate _next;
+    private readonly IWebHostEnvironment _env;
+
+    public DynamicJavaScriptMiddleware(RequestDelegate next, IWebHostEnvironment env)
+    {
+        _next = next;
+        _env = env;
+    }
```

Comment:
```suggestion
public class DynamicJavaScriptMiddleware(RequestDelegate next, IWebHostEnvironment env)
{
```
You can replace this boilerplate if you want to be trendy. This is called a [primary constructor](https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/primary-constructors). Fair warning: some people have strong opinions against these since you can't currently enforce readonly.